redbot/cogs/streams/streamtypes.py

```diff
@@ -36,10 +36,14 @@ def __init__(self, **kwargs):
         self.channels = kwargs.pop("channels", [])
         self._messages_cache = kwargs.pop("_messages_cache", [])
         self._token = kwargs.pop("token", None)
+        self.token_name = "twitch"
```

Comment:
Since token_name is a constant, it's best to declare it as a class variable:
```py
class TwitchCommunity:

    token_name = "twitch"

    def __init__(self, **kwargs):
```